src/extra/AutoUi/Collection/NoRecordsFoundArrow.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import styled from 'styled-components';
+import { useTheme } from '../../../hooks/useTheme';
+import { Flex } from '../../../components/Flex';
+import { Txt } from '../../../components/Txt';
+
+const Left = styled.div`
+	padding-left: 58px;
+	padding-bottom: 16px;
+	pointer-events: none;
+`;
+
+const ContentWrapper = styled(Flex)`
+	pointer-events: none;
+`;
+
+export const NoRecordsFoundArrow = ({
+	children,
+}: {
+	children: React.ReactNode;
+}) => {
+	const theme = useTheme();
+	return (
+		<Flex mt={-3} className="onboarding__no-records-found">
```

Comment:
```suggestion
		<Flex mt={-3}>
```
I guess we no longer need that, but tbh I didn't check the e2e